python-threatexchange/threatexchange/exchanges/clients/stopncii/api.py

```diff
@@ -226,11 +229,25 @@ def fetch_hashes(
         logging.debug("StopNCII FetchHashes called: %s", params)
         json_val = self._get(StopNCIIEndpoint.FetchHashes, **params)
         logging.debug("StopNCII FetchHashes returns: %s", json_val)
-        return dacite.from_dict(
+        records: t.List[StopNCIIHashRecord] = []
```

Comment:
The logic below seems reasonable. Can you just add a short comment explaining why it's not a straight convert?

Maybe something like
```
# If there is a malformed record or a change that would prevent the deserialization
# of a record, skip over it instead of crashing. Please open an issue if you see the logging
# statement below.
```